multiqc/modules/bcftools/stats.py

```diff
@@ -31,6 +31,10 @@ def parse_bcftools_stats(self):
 
         self.bcftools_stats = dict()
         self.bcftools_stats_indels = dict()
+        self.bcftools_stats_variant_quality_count_snp = dict()
```

Comment:
I think that these can probably just be called `self.bcftools_stats_vqc_snp` etc.. ;)